### PR TITLE
Feature/fetch user contacts

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]
see comments

File: package-lock.json
[the remark this answers is not in the document]
Comment (Le-Caignec) [reply package-lock.json:1]: Commit: 65bf307e84da2c0dc008880598d0c7526897537f

File: src/web3mail/types.ts [hunk 1]
```diff
@@ -26,6 +26,9 @@ export type SendEmailResponse = {
   taskId: Address;
 };
 
+export type FetchUserContactsParams = {
+  address: Address;
```
[the remark this answers is not in the document]
Comment (Le-Caignec) [reply src/web3mail/types.ts]: Commit: dc8c1ab51595001e3de8e4cce2c917db81148d6b

**Review by PierreJeanjacquot — APPROVED** [review]
👍

**Pull request closed without merging**